terraform/modules/single_instance/data.tf

```diff
@@ -3,14 +3,16 @@ locals {
   any_enabled  = var.nfs_enabled || var.nginx_enabled
   ports = {
     ssh  = jsondecode(shell_script.port_ssh.output["ports"])[0]
-    http = var.nginx_enabled ? jsondecode(shell_script.port_http[0].output["ports"])[0] : null
+    http = var.nginx_enabled ? ( var.alt_http ? jsondecode(shell_script.port_http[0].output["ports"])[0] : 80 ) : null
+    https = var.nginx_enabled ? ( var.alt_http ? jsondecode(shell_script.port_http[0].output["ports"])[1] : 443) : null
```

Comment:
we should add this in the current docs if we do not have it yet, ports 80 and 443 are open from ugent net